### DISCUSSION TITLE
Hello :)

### DISCUSSION THREAD

**Discussion by goldjacobe** [GlitchTech-Developments/GraphiteDesktop#5] [Ideas]
want some help with this? :p 

we actually started working on an electron version a little while ago that we might do more with soon — https://github.com/withgraphite/graphite-desktop/releases/tag/v0.1.3

**Comment by JesseKoldewijn** [comment]
Feel free to contribute, I choose to build a simple starting setup using Tauri instead of Electron mainly due to performance and hardware requirements and such. 

I've heard from you guys through a reply on one of my tweets that you guys seemed interested in a desktop app. Which got me the idea for this project. 

My twitter/X is: https://x.com/dull_joker

**Comment by JesseKoldewijn** [comment]
@goldjacobe I just had a further look into the electron version you guys build and I think the one I've build might have a slighty better setup with regards to performance, error handling and application executable size. 

If you guys want me to help with the project, feel free to reach out!
